### ISSUE TITLE
Does this support next/future/image?

### ISSUE THREAD

**Issue by ghost** [Niels-IO/next-image-export-optimizer#24]
If not, can you add support for it. https://nextjs.org/docs/api-reference/next/future/image

**Comment by Niels-IO** [comment]
Hi @robert-sperse, the new image api from Next.js is not yet supported. I will have a look at it

**Comment by milkeddy1** [comment]
Hi, I use the component ExportImage but it come to an error says "filePath.split is not a function", how do i fix it?

**Comment by elliott-w** [comment]
Looks like `next/future/image` is now stable: https://nextjs.org/blog/next-12-3#image-component

**Comment by Niels-IO** [comment]
Hi @elliott-w,

Thanks for bringing awareness to their progress with the new Image component. Once they announce their next major version and more implementation details, I will reflect the changes here so that the syntax matches closely.

**Comment by elliott-w** [comment]
Forgive my ignorance, but I can't see what further implementation details you'd need than those specified in the [api reference here](https://nextjs.org/docs/api-reference/next/future/image)?

**Comment by RichardLindhout** [comment]
Would be really nice if the future image is supported since it's a lot less 'hacky' than the old component with spans e.t.c.

**Comment by Niels-IO** [comment]
Hey, definitely, the new implementation is a lot simpler. I am on it to support the future image as well.

**Comment by RichardLindhout** [comment]
Changing is fairly easy
https://github.com/RichardLindhout/next-image-export-optimizer/commit/3972110868c10c3b55959e9363a6475f7df0fcff
https://github.com/RichardLindhout/next-image-export-optimizer/commit/5a616c064416b68880f4b797f043ee08c7455fb6

If you can't wait you could use my fork :)
```
yarn add https://github.com/RichardLindhout/next-image-export-optimizer
```

**Comment by Niels-IO** [comment]
I just published v0.17.0, which supports the next/future/image component. 

Please let me know whether it works as expected, as I had to implement the blurry placeholder image using an SVG filter since the wrapper that had overflow:hidden set (which limits the blur to not extend over the image borders) is gone in the next/future/image component.
